### PR TITLE
Add beta support

### PR DESCRIPTION
**Pull request by domdomegg**
Required for https://github.com/commons-app/apps-android-commons/issues/2793

Think it'd be best to move the prod version to its own directory - annoyingly this means we'll have two copies of the same file for backwards compatibility temporarily until the new update is out and everyone has updated. I did try a symlink but as it's not really truly hosted we can't get 301 redirects/similar.

### CODE REVIEW

File: prod/campaigns.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (nicolas-raoul) [inline prod/campaigns.json]: Is it really worth moving this file here?
This means we will have to maintain both for a while, until everybody uses the new app version, right?

**Comment by misaochan** [comment]
Sorry if I'm missing something, but why not create a new directory for beta, instead of a new one for prod?

**Comment by ashishkumar468** [comment]
Agree with @misaochan. We can maybe create and add another file, campaigns_beta.json instead of moving the original file (so that we donot have to maintain both two versions of campaigns prod)

File: prod/campaigns.json (hunk 1 above)
Comment (maskaravivek) [reply prod/campaigns.json]: I agree with @nicolas-raoul! We can keep using the existing file for prod campaigns and have a new file for the beta.

**Comment by domdomegg** [comment]
Thanks for all the feedback! I understand issue with moving directory, agree with changes requested and will make them later today.

I think it'd be useful to have two beta ones; one with a sample campaign and one with no campaigns. This would allow testers to test for both cases by swapping them in the build config. Trying to think of maybe better names for them - maybe `campaigns_beta_positive.json` and `campaigns_beta_negative.json`. I'm planning on putting these in the top directory (next to the current `campaigns.json`). Any ideas/feedback on this appreciated. :)

**Comment by misaochan** [comment]
Thanks @domdomegg !

**Pull request merged**